packages/dx-grid-core/src/plugins/integrated-grouping/computeds.ts

```diff
@@ -27,7 +27,7 @@ export const groupedRows: GroupedRowsFn = (
   const { columnName } = grouping[0];
   const groupCriteria = (getColumnCriteria && getColumnCriteria(columnName))
     || defaultColumnCriteria;
-  const groups = rows
+  const groups = Array.prototype.slice.call(rows)
```

Comment:
Is it required?

Reply:
Without these changes, a demo will break down.